tracer/src/Datadog.Trace/Propagators/W3CTraceContextPropagator.cs

```diff
@@ -157,7 +157,7 @@ internal static string CreateTraceStateHeader(SpanContext context)
                 }
 
                 // origin ("o:<value>")
-                var origin = context.TraceContext?.Origin;
+                var origin = context.Origin;
```

Comment:
For other reviewers, `SpanContext.Origin` returns `TraceContext?.Origin ?? _origin`, so this makes sure to return the origin when we don't have a TraceContext, which is possible when the SpanContext represents a remote parent.